pkg/storage/chunk/client/azure/blob_storage_client_test.go

```diff
@@ -92,3 +93,29 @@ func Test_Hedging(t *testing.T) {
 		})
 	}
 }
+
+func Test_EndpointSuffixWithContainer(t *testing.T) {
```

Comment:
This looks great, but it only tests the new changes - can you also add a test to validate that the default address still works please?

Reply:
updated

Reply:
Thank you